OpenRA.Mods.Cnc/Traits/TDGunboat.cs

```diff
@@ -206,6 +206,8 @@ public void SetPosition(Actor self, WPos pos)
 		public Activity ReturnToCell(Actor self) { return null; }
 		public Activity MoveToTarget(Actor self, in Target target,
 			WPos? initialTargetPosition = null, Color? targetLineColor = null) { return null; }
+		public Activity MoveOntoTarget(Actor self, in Target target, in WVec offset,
+			WAngle? facings, Color? targetLineColor = null) { return null; }
```

Comment:
Why plural `facings`?

Reply:
I suppose it could be changed to singular. Sometime distant time the future we will want to make this support multiple facings.

Reply:
We should change the name when we do. At the moment it is just confusing.

Reply:
Seems like the implementations already use the singular anyway.

Reply:
Why was this marked as resolved again, is it really that much effort to change this to match the implementations and what the code actually does?

Reply:
because I resolved it?

Reply:
ah I see. I fixed it everywhere except for TD gunboat